examples/example_multi_genome.py

```diff
@@ -0,0 +1,140 @@
+"""
+Example for evolutionary regression with multiple genomes
+=========================================================
+
+Example demonstrating the use of Cartesian genetic programming with multiple
+genomes per individual for a simple regression task with a piecewise
+target function.
+"""
+
+# The docopt str is added explicitly to ensure compatibility with
+# sphinx-gallery.
+docopt_str = """
+   Usage:
+     example_multi_genome.py [--max-generations=<N>]
+
+   Options:
+     -h --help
+     --max-generations=<N>  Maximum number of generations [default: 300]
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy.constants
+from docopt import docopt
+
+import cgp
+
+args = docopt(docopt_str)
+
+# %%
+# We first define a target function.  The function applies different
+# transformations to the input depending whether the input is less or greater
+# than zero. We thus need to fit two different functions.
```

Comment:
```suggestion
# transformations to the input depending whether the input is less than zero or not.
# We thus need to fit two different functions.
```
or some other formulation to include "equal to zero"

Reply:
Thanks done.